xml/homepage.xml

```diff
@@ -73,6 +73,14 @@ Enterprise distributions, commercial support and training are
   description="NGINX Ingress Controller connects Kubernetes apps and services with rock solid request handling, auth, self-service CRDs, and easy debugging."
 />
 
+<project
+  title="Gateway Fabric"
+  docs="https://docs.nginx.com/nginx-gateway-fabric/"
+  repo="https://github.com/nginxinc/nginx-gateway-fabric"
+  logo="img/gateway_fabric_logo.svg"
+  description="NGINX Gateway Fabric provides an implementation for the Gateway API using NGINX as the data plane."
```

Comment:
Some mention of Kubernetes would be useful. Gateway API is not a universally recognised term.